src/commands/synthetics/tunnel.ts

```diff
@@ -169,11 +169,20 @@ export class Tunnel {
           })
         })
         dest.on('error', (error) => {
-          console.error('Forwarding error', error.message)
+          if (!src) {
+            if ((error as any).code === 'ENOTFOUND') {
```

Comment:
Nit: I'd argue to use `'code' in error && error.code === 'ENOTFOUND'` though I do not feel strongly about this

Reply:
The typings prevented that but it seems we just needed to specify the error parameter as `NodeJS.ErrnoException`.